### ISSUE TITLE
Another fantastic update!

### ISSUE THREAD

**Issue by eisnerguy1** [jorio/OttoMatic#1]
Thank you for brining another great PangeaSoft game to modern OS's!

**Comment by orazioedoardo** [comment]
This is amazing @jorio! I was experimenting with the iOS version of the apps on Mac but this is on another level. For me, the funniest part is hacking the source code to include cheats, thank you for this work and thanks Pangea Software for sharing the source code.

**Comment by fordcars** [comment]
Absolutely insane. @jorio did it again!

**Comment by jpenguin** [comment]
Wow, another part of my childhood!

Linux builds https://build.opensuse.org/package/binaries/home:jpenguin:Games/Ottomatic/AppImage

**Comment by jorio** [comment]
@jpenguin Good idea! I'll create an itch.io page to gather my ports in one place one of these days :)

**Comment by foote-darrell** [comment]
I can finally update my iMacs. The reason I have not updated it to Catalina was just because it would not play _Otto Matic_, among with many other 32-bit applications. Did you have access to the actual, raw Maya files?

**Comment by foote-darrell** [comment]
Can you do _Weekend Warrior_ too? It is underrated game that should get more exposure.

**Comment by jpenguin** [comment]
@jorio
Done already actually, send me an itch username & I'll mark you as the developer

@foote-darrell

I'm surprised too that Brian sent the source for Otto before weekend, cromag, & firefall

**Comment by jorio** [comment]
@foote-darrell Depending on what you want to do with the models, you might be able to use the [3DMF character models](https://github.com/jorio/OttoMatic/tree/master/3DMF). You can probably edit those with an ancient version of Form\*Z, I guess.

Pangea isn’t eager to have WW ported. I respect that and won’t be able to do that one. Sorry!

There’s no particular order for doing these ports, by the way. I wanted to do Otto because it’s one of my favorites :)

**Comment by foote-darrell** [comment]
@jorio You mean they just aren't eager to get _Weekend Warrior_ ported as an understatement? Or they actually want it to _not_ get remastered? There are many people who appreciate the game; I am one of them. it is featured on my [list of greatest video games.](https://telegra.ph/My-list-of-greatest-video-games-08-17). The game does exist. Are they not wanting to renew the existence of the game anymore? Cancel culture is at it again!

**Comment by Smashman4** [comment]
Thank you for this port. I bought it on iOS not too long ago, but would rather play it on my laptop/linux devices. Curious how you feel about Cro-Mag Rally. I always loved that racing game.

**Comment by jorio** [comment]
@Smashman4 the CMR port is in progress
![image](https://user-images.githubusercontent.com/837086/132138124-349238d9-05c1-44f0-a17f-0a1bc8a56b5a.png)

**Comment by Smashman4** [comment]
> @Smashman4 the CMR port is in progress
> ![image](https://user-images.githubusercontent.com/837086/132138124-349238d9-05c1-44f0-a17f-0a1bc8a56b5a.png)

OH WOW! Thank you so much. My childhood is coming back again.
